messages/no-config-found.txt

```diff
@@ -0,0 +1,21 @@
+We couldn't find a valid configuration file for this linting run.
+
+This is the directory we were trying to lint: <%= directory %>
```

Comment:
It might be slightly more accurate to say something like `We looked for configuration files in <%= directory =%> and its ancestors.`